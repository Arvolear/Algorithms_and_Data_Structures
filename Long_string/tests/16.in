70393402119677
900067862868792282598990220819010218915313