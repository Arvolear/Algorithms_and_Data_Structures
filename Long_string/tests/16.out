900067862868792282598990220889403621034990
900067862868792282598990220748616816795636
63358839005921190028036777412780708239170317933953913901
12786253196550606529791006309